dev-qt/qtmultimedia/qtmultimedia-6.5.2.ebuild

```diff
@@ -34,30 +33,45 @@ RDEPEND="
 	)
 	gstreamer? (
 		dev-libs/glib:2
-		media-libs/gstreamer:1.0
 		media-libs/gst-plugins-bad:1.0
 		media-libs/gst-plugins-base:1.0
+		media-libs/gstreamer:1.0
 		media-libs/libglvnd
 	)
 	pulseaudio? ( media-libs/libpulse[glib] )
 	vaapi? (
-		=dev-qt/qtbase-${PV}*[opengl]
+		=dev-qt/qtbase-${PV}*:6[opengl]
 		media-libs/libglvnd
 		media-libs/libva:=
 	)
 "
-DEPEND="${RDEPEND}
+DEPEND="
+	${RDEPEND}
 	gstreamer? ( x11-base/xorg-proto )
 	v4l? ( sys-kernel/linux-headers )
 "
 
+CMAKE_SKIP_TESTS=(
+	# tries to use real alsa or pulseaudio and fails in sandbox
+	tst_qaudiosink
+	tst_qaudiosource
+	tst_qmediacapturesession
+	tst_qmediaplayerbackend
+	tst_qsoundeffect
+	# may try to use hardware acceleration for encoding
+	tst_qscreencapture_integration
+)
+
 src_configure() {
+	# normally passed by build system, but needed for 32-on-64 chroots
+	use x86 && append-cppflags -DPFFFT_SIMD_DISABLE
```

Comment:
ha, I googled it because I was wondering wtf it stood for ("FFT", ok, but the rest?) and came across https://chromium.googlesource.com/chromium/src/+/refs/heads/main/third_party/pffft/patches/05-fix-arch-detection.diff.